bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,837 +4,298 @@
 	</channel-type>
 
-	<channel-type id="location">
-		<item-type>Location</item-type>
-		<label>Location</label>
-		<description>Location of the device</description>
-		<state readOnly="true" pattern="%2$s°N,%3$s°W, %1$s m"/>
+	<channel-type id="contact">
+		<item-type>Contact</item-type>
+		<label>A contact representing on/off or open/closed</label>
+		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="temperature">
-		<item-type>Number:Temperature</item-type>
-		<label>Temperature</label>
-		<description>Current temperature</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
+	<channel-type id="snapshot">
+		<item-type>Image</item-type>
+		<label>Event Snapshot</label>
+		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="minTemp" advanced="true">
-		<item-type>Number:Temperature</item-type>
-		<label>Min Temp</label>
-		<description>Minimum Temperature on current day</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
+	<channel-type id="camera-event">
+		<kind>trigger</kind>
+		<label>Camera Event</label>
+		<event>
+			<options>
+				<option value="ANIMAL"/>
+				<option value="HUMAN"/>
+				<option value="MOVEMENT"/>
+				<option value="VEHICLE"/>
+			</options>
+		</event>
 	</channel-type>
 
-	<channel-type id="minTempThisWeek" advanced="true">
-		<item-type>Number:Temperature</item-type>
-		<label>Min Temp This Week</label>
-		<description>Minimum Temperature this week</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
+	<channel-type id="string-ro">
+		<item-type>String</item-type>
+		<label>Read-Only String</label>
+		<state readOnly="true"/>
 	</channel-type>
 
-	<channel-type id="minTempThisMonth" advanced="true">
-		<item-type>Number:Temperature</item-type>
-		<label>Min Temp This Month</label>
-		<description>Minimum Temperature this month</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
+	<channel-type id="string-rw">
+		<item-type>String</item-type>
+		<label>Read-Write String</label>
+		<state readOnly="false" pattern="%s"/>
 	</channel-type>
 
-	<channel-type id="maxTemp" advanced="true">
-		<item-type>Number:Temperature</item-type>
-		<label>Max Temp</label>
-		<description>Maximum Temperature on current day</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%"/>
+	<channel-type id="person-count">
+		<item-type>Number</item-type>
+		<label>Person Count</label>
+		<tags>
+			<tag>Status</tag>
+			<tag>Presence</tag>
+		</tags>
+		<state readOnly="true" pattern="%d"/>
 	</channel-type>
 
-	<channel-type id="maxTempThisWeek" advanced="true">
-		<item-type>Number:Temperature</item-type>
-		<label>Max Temp This Week</label>
-		<description>Maximum Temperature this week</description>
-		<category>Temperature</category>
+	<channel-type id="rain-qtty">
```

Comment:
It would be clearer if this was `rain-quantity` instead of `rain-qtty`

Reply:
You're correct.